requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==2.1.10
```

Comment:
If I remember correctly, this backend will depend on `django-netjsonconfig`, which will choose the correct django version, just a note for you to keep in mind, if we later add more dependencies here we can remove these ones to avoid redundancy

Reply:
please what will be the use of `django_netjsonconfig` repo here? i thought we will use `openwisp_controller` for the creation of templates since it already has `template-organization` relationship which `django_netjsonconfig` doesn't have.

Reply:
:+1: